microsoftgraph/microsoft-graph-explorer-v4
# Capture telemetry for Graph Proxy API calls

## Overview

The current configuration only whitelists calls to the endpoint  for fetching the proxy URL. 

We need to capture calls to the proxy API .

## Testing Instructions
- Open developer tools on browser and go to network tab
- Run query as an anonymous user
- Check to see if telemetry data has been sent to Application Insights
![image](https://user-images.githubusercontent.com/25274795/132318053-ffe59edb-0fe1-4120-b1f2-5c6a9b27fab0.png)
